lib/content-api-client.js

```diff
@@ -0,0 +1,99 @@
+require('colors');
+const NetworkUtils = require('./utils/NetworkUtils');
+const {
+    renderedRegionsByPageTypeQuery,
+    renderedRegionsByPageTypeAndEntityIdQuery,
+} = require('./graphql/query');
+
+const networkUtils = new NetworkUtils();
+
+/**
+ * @param {object} options
+ * @param {string} options.accessToken
+ * @param {string} options.storeUrl
+ * @param {string} pageType
+ * @returns {Promise<renderedRegions: array}>}
+ */
+async function getRenderedRegionsByPageType({ accessToken, storeUrl, pageType }) {
+    try {
+        const query = renderedRegionsByPageTypeQuery(pageType);
+
+        const response = await networkUtils.sendApiRequest({
+            url: `${storeUrl}/graphql`,
+            headers: {
+                'cache-control': 'no-cache',
+                'content-type': 'application/json',
+                Authorization: `Bearer ${accessToken}`,
+            },
+            method: 'POST',
+            data: JSON.stringify({
+                query,
+            }),
+        });
+
+        const {
+            site: { content },
+        } = response.data.data;
+        const {
+            renderedRegionsByPageType: { regions },
+        } = content;
+
+        return {
+            renderedRegions: regions,
+        };
+    } catch (err) {
+        console.log(err);
+        throw err;
+    }
+}
+
+/**
+ * @param {object} options
+ * @param {string} options.accessToken
+ * @param {string} options.storeUrl
+ * @param {string} pageType
+ * @param {number} entityId
+ * @returns {Promise<renderedRegions: array}>}
+ */
+async function getRenderedRegionsByPageTypeAndEntityId({
+    accessToken,
+    storeUrl,
+    pageType,
+    entityId,
+}) {
+    try {
+        const query = renderedRegionsByPageTypeAndEntityIdQuery(pageType, entityId);
+
+        const response = await networkUtils.sendApiRequest({
+            url: `${storeUrl}/graphql`,
+            headers: {
+                'cache-control': 'no-cache',
+                'content-type': 'application/json',
+                Authorization: `Bearer ${accessToken}`,
+            },
+            method: 'POST',
+            data: JSON.stringify({
+                query,
+            }),
+        });
+
+        const {
+            site: { content },
+        } = response.data.data;
+        const {
+            renderedRegionsByPageTypeAndEntityId: { regions },
+        } = content;
+
+        return {
+            renderedRegions: regions,
+        };
+    } catch (err) {
+        console.log(err);
```

Comment:
`console.error`
It's also possible to highlight error with red color like here https://github.com/bigcommerce/stencil-cli/blob/master/lib/release/release.js#L145